tests/suites/1_stateful/05_formats/05_05_parquet/05_05_01_parquet_load_unload.result

```diff
@@ -0,0 +1,41 @@
+>>>> drop table if exists test_load_unload
+>>>> CREATE TABLE test_load_unload
+ (
+     a VARCHAR NULL,
+     b float,
+     c array(string not null),
+     d Variant,
+     e timestamp,
+     f decimal(4, 2),
+     g tuple(string not null,int not null),
+     h array(variant),
+     i array(tuple(string, int))
+ );
+>>>> drop stage if exists s1
+>>>> create stage s1 url='fs:///tmp/test_load_unload/'
+>>>> truncate table test_load_unload
+>>>> insert into test_load_unload values
+ 	('a"b', 1, ['a"b'], parse_json('{"k":"v"}'), '2044-05-06T03:25:02.868894-07:00', 010.011, ('a', 5),
```

Comment:
json value can be auto parsed, we can remove `parse_json` function.

Reply:
thanks, fixed, plz review again